superset-frontend/src/CRUD/CollectionTable.tsx

```diff
@@ -181,15 +205,75 @@ export default class CRUDCollection extends React.PureComponent<
     }));
   }
 
+  sortColumn(col: string, sort = 0) {
+    const { sortColumns } = this.props;
+    // default sort logic sorting string, boolean and number
+    const compareSort = (m: any, n: any) => {
```

Comment:
If the comparator can only handle `string`, `boolean` and `number`, maybe we can add these types instead of `any`.